day_01.md

```diff
@@ -30,7 +29,7 @@ A great overview regarding ECMAScript and JavaScript.
 
 Let's start with some facts
 
-- In 1995 [Brendan Eich](https://en.wikipedia.org/wiki/Brendan_Eich) was recruited by [Netscape](https://en.wikipedia.org/wiki/Netscape_Navigator) to embed [Scheme](https://en.wikipedia.org/wiki/Scheme_(programming_language)) programming language into netscape Navigator. Due to different company level decisions the original idea mutated, and it ends up with Brendan writing a new language prototype written in 10 days,
+- In 1995 [Brendan Eich](https://en.wikipedia.org/wiki/Brendan_Eich) was recruited by [Netscape](https://en.wikipedia.org/wiki/Netscape_Navigator) to embed [Scheme](<https://en.wikipedia.org/wiki/Scheme_(programming_language)>) programming language into netscape Navigator. Due to different company level decisions, the original idea mutated, and it ends up with Brendan writing a new language prototype written in 10 days,
```

Comment:
Please check surrounding < and > for URLs inside markdown hyperlink notation. (was it suggested by a linter?)
Verify this everywhere it's been added.